lib/hammer_cli_foreman_ssh/ssh.rb

```diff
@@ -12,6 +12,10 @@ class Command < HammerCLIForeman::Command
 
     command_name 'SSH to hosts'
     option %w(-c --command), 'COMMAND', _('Command to execute'), :attribute_name => :command, :required => true
+    option %w(-n --concurrent), 'CONCURRENCY', _('Number of concurrent SSH sessions'), :attribute_name => :concurrent do |o|
+      Integer(o)
+      o.nil? ? o : o.to_i
```

Comment:
this line is not neccessary, as the block is executed only if the parameter was set